Modules/ITS/src/ITSTrackCheck.cxx

```diff
@@ -129,18 +123,24 @@ Quality ITSTrackCheck::check(std::map<std::string, std::shared_ptr<MonitorObject
 
     if (iter->second->getName() == "VertexRvsZ") {
       auto* h = dynamic_cast<TH2D*>(iter->second->getObject());
-      TH1D* projectZ = h->ProjectionY();
-      if ((projectZ->Integral(1, projectZ->FindBin(-10)) > 0) || (projectZ->Integral(projectZ->FindBin(10), projectZ->GetNbinsX()) > 0))
-        result = result.getLevel() + 1e6;
-
+      result.set(Quality::Good);
+      result.addMetadata("CheckRZVertexRDisplacedBad", "good");
+      result.addMetadata("CheckRZVertexZDisplacedBad", "good");
       TH1D* projectR = h->ProjectionX();
-      if (projectR->Integral(projectR->FindBin(0.3), projectR->GetNbinsX()) > 0)
-        result = result.getLevel() + 2e6;
+      TH1D* projectZ = h->ProjectionY();
+      if (projectR->Integral(projectR->FindBin(0.3), projectR->GetNbinsX()) > 0) {
```

Comment:
This check can be removed because the `o2-its-reco-workflow` gives us already vertices which stay within the beampipe (and which can be of course with R>0.3).

Reply:
please check my corrections, thank you.